src/integration/car/ev-database/EVDatabaseCarIntegration.ts

```diff
@@ -39,54 +40,96 @@ export default class EVDatabaseCarIntegration extends CarIntegration {
+            chargePhaseVolt: chargeStandardTableUniversal[chargeStandard].Charge_PhaseVolt,
+            chargePhaseAmp: chargeStandardTableUniversal[chargeStandard].Charge_PhaseAmp,
+            chargePhase: chargeStandardTableUniversal[chargeStandard].Charge_Phase,
+            chargePower: chargeStandardTableUniversal[chargeStandard].Charge_Power,
+            chargeTime: chargeStandardTableUniversal[chargeStandard].Charge_Time,
+            chargeSpeed: chargeStandardTableUniversal[chargeStandard].Charge_Speed,
           };
-          chargeAlternativeTables.push(chargeAlternativeTable);
+          chargeStandardTables.push(chargeStandardTable);
         }
-      }
-      if (data.Charge_Option_Table) {
-        for (const chargeOption of Object.keys(data.Charge_Option_Table)) {
-          const chargeAlternativeTable: CarCatalogConverter = {
-            type: chargeOption,
-            evsePhaseVolt: data.Charge_Standard_Table[chargeOption].EVSE_PhaseVolt,
-            evsePhaseAmp: data.Charge_Standard_Table[chargeOption].EVSE_PhaseAmp,
-            evsePhase: data.Charge_Standard_Table[chargeOption].EVSE_Phase,
-            chargePhaseVolt: data.Charge_Standard_Table[chargeOption].Charge_PhaseVolt,
-            chargePhaseAmp: data.Charge_Standard_Table[chargeOption].Charge_PhaseAmp,
-            chargePhase: data.Charge_Standard_Table[chargeOption].Charge_Phase,
-            chargePower: data.Charge_Standard_Table[chargeOption].Charge_Power,
-            chargeTime: data.Charge_Standard_Table[chargeOption].Charge_Time,
-            chargeSpeed: data.Charge_Standard_Table[chargeOption].Charge_Speed,
-          };
-          chargeOptionTables.push(chargeAlternativeTable);
+        if (data.Charge_Alternative_Table) {
+          for (const chargeAlternative of Object.keys(data.Charge_Alternative_Table)) {
+            const chargeAlternativeTable: CarCatalogConverter = {
+              type: chargeAlternative,
+              evsePhaseVolt:chargeStandardTableUniversal[chargeAlternative].EVSE_PhaseVolt,
+              evsePhaseAmp: chargeStandardTableUniversal[chargeAlternative].EVSE_PhaseAmp,
+              evsePhase: chargeStandardTableUniversal[chargeAlternative].EVSE_Phase,
+              chargePhaseVolt: chargeStandardTableUniversal[chargeAlternative].Charge_PhaseVolt,
+              chargePhaseAmp: chargeStandardTableUniversal[chargeAlternative].Charge_PhaseAmp,
+              chargePhase: chargeStandardTableUniversal[chargeAlternative].Charge_Phase,
+              chargePower: chargeStandardTableUniversal[chargeAlternative].Charge_Power,
+              chargeTime: chargeStandardTableUniversal[chargeAlternative].Charge_Time,
+              chargeSpeed: chargeStandardTableUniversal[chargeAlternative].Charge_Speed,
+            };
+            chargeAlternativeTables.push(chargeAlternativeTable);
+          }
+        }
+        if (data.Charge_Option_Table_DE) {
```

Comment:
Here you prioritise DE > UK > NL (not the same as standard table)

Why don't you build the chargeOptionTables using the 3 countries and make sure that we have unique  chargeOption ? This would render the logique less dodgy and would handle the case where an option was set for one country but not another ?

Also: code duplication create a method

Reply:
Done, now the standard table and the option table is created using 3 the 3 countries table.
Priority: DE > NL > UK. 
Also I created a method to avoid code duplication.